velox/docs/functions/spark/string.rst

```diff
@@ -192,6 +192,17 @@ String Functions
 
         SELECT ltrim('ps', 'spark'); -- "ark"
 
+.. spark:function:: luhn_check(string) -> boolean
+
+    Returns true if ``string`` passes the Luhn algorithm check. Otherwise, returns false.
+    This checksum function is widely applied on credit card numbers and government identification
```

Comment:
> This checksum function is widely applied on credit card numbers and government identification numbers to distinguish valid numbers from mistyped, incorrect numbers.

Perhaps focus more on what is the Luhn algorithm, rather than where it is adopted. Consider the introduction like below:

> The Luhn algorithm is a simple check digit formula used to validate a variety of identification numbers, defined in ...

Reply:
Provide definition source, US patent 2950048A.